AdobeStockClient/Test/Unit/Model/StockFileToDocumentTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\AdobeStockClient\Test\Unit\Model;
+
+use AdobeStock\Api\Models\StockFile;
+use Magento\AdobeStockClient\Model\StockFileToDocument;
+use Magento\Framework\Api\AttributeValueFactory;
+use Magento\Framework\Api\Search\DocumentFactory;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\LoggerInterface;
+
+class StockFileToDocumentTest extends TestCase
+{
+    /**
+     * @var ObjectManager
+     */
+    private $objectManager;
+
+    /**
+     * @var StockFileToDocument|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $stockFileToDocument;
+
+    /**
+     * @var DocumentFactory
+     */
+    private $documentFactory;
+
+    /**
+     * @var AttributeValueFactory
+     */
+    private $attributeValueFactory;
+
+    /**
+     * @inheritdoc
+     */
+    public function setUp(): void
+    {
+        $this->objectManager = new ObjectManager($this);
+
+        $this->documentFactory = $this->getMockBuilder(DocumentFactory::class)
+            ->setMethods([])
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->attributeValueFactory = $this->getMockBuilder(AttributeValueFactory::class)
+            ->setMethods([])
+            ->disableOriginalConstructor()
+            ->getMock();
+        $logger = $this->getMockBuilder(LoggerInterface::class)
+            ->setMethods([])
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        $this->stockFileToDocument = $this->objectManager->getObject(
+            StockFileToDocument::class,
+            [
+                'documentFactory' => $this->documentFactory,
+                'attributeValueFactory' => $this->attributeValueFactory,
+                'logger' => $logger,
+            ]
+        );
+    }
+
+    public function testConvert()
+    {
+        /** @var StockFile $stockFile */
+        $stockFile = new StockFile([]);
+
+        $stockFileId = 5;
```

Comment:
Can you please test a bit more information as part of `$stockFile` like 'url', 'title', 'creator_name', 'keywords' array. It would be great if you could provide the stock file and array of expected `attribute values` via data provider.